install/updates/froxlor/0.10/update_0.10.inc.php

```diff
@@ -263,3 +263,9 @@
 if (\Froxlor\Froxlor::isFroxlorVersion('0.10.0-rc1')) {
 	\Froxlor\Froxlor::updateToVersion('0.10.0-rc2');
 }
+
+if (\Froxlor\Froxlor::isDatabaseVersion('201904250')) {
+
+	Database::query("ALTER TABLE `" . TABLE_PANEL_DOMAINS . "` ADD `caa` text default NULL AFTER `letsencrypt`;");
```

Comment:
please also add an showUpdateStep() and lastStepStatus() as we always do

Reply:
fixed in b427212b00fd708b2969a6ed14fc410f69f422ff.